src/simsopt/geo/surfacerzfourier.py

```diff
@@ -375,6 +382,36 @@ def from_focus(cls, filename, **kwargs):
         surf.local_full_x = surf.get_dofs()
         return surf
 
+    @classmethod
+    @SimsoptRequires(Qsc is not None, "from_pyQSC method requires pyQSC module")
+    def from_pyQSC(cls, stel: Qsc, r: float = 0.1, ntheta=20, mpol=10, ntor=20, **kwargs):
+        """
+        Initialize the surface from a pyQSC object. This creates a surface
+        from a near-axis equilibrium with a specified minor radius `r` (in meters).
+
+        Args:
+            stel: Qsc object with a near-axis equilibrium.
+            r: the near-axis coordinate radius (in meters).
+            kwargs: Any other arguments to pass to the ``SurfaceRZFourier`` constructor.
```

Comment:
Document `ntheta`, `mpol`, and `ntor` in docstring.

Reply:
Fixed in commit https://github.com/hiddenSymmetries/simsopt/pull/306/commits/fe9697e005c3fc94758b9f257e62b802cf38ceb7